vars/treebuild.groovy

```diff
@@ -3,9 +3,9 @@
  * @param the file with a collection of items following the pattern PROJECT_GROUP/PROJECT_NAME, for example kiegroup/drools
  * @param settingsXmlId maven settings xml file id
  * @param goals maven goals
- * @param skipTests boolean to skip tests or not
+ * @param skipTests Boolean to skip tests or not
  */
-def downstreamBuild(def projectCollection, String settingsXmlId, String goals, boolean skipTests) {
+def downstreamBuild(def projectCollection, String settingsXmlId, String goals, Boolean skipTests = null) {
```

Comment:
why not to use false/true instead of null? more readable, as null = false

Reply:
because due to `jbpm-services-ejb` where we have a profile like this
```
<profile>
      <id>ejb-war</id>
      <activation>
        <property>
          <name>skipTests</name>
          <value>false</value>
        </property>
      </activation>
```
we have have three different cases for `skipTest`

- `skipTests` or `skipTests=true`
- `skipTests=false`
- `skipTests` not present

so it has to be represented by true, false or null

- 

-

Reply:
ok